sw/device/boot_rom/bootstrap.c

```diff
@@ -21,7 +24,15 @@ static int bootstrap_requested(void) {
     return !!(REG32(FLASH_MEM_BASE_ADDR) == 0 ||
               REG32(FLASH_MEM_BASE_ADDR) == 0xFFFFFFFF);
   }
-  return !!(gpio_read() & GPIO_BOOTSTRAP_BIT_MASK);
+  // Initialize GPIO device
+  dif_gpio_t gpio;
+  dif_gpio_config_t gpio_config = {.base_addr =
+                                       mmio_region_from_addr(GPIO0_BASE_ADDR)};
+  dif_gpio_init(&gpio_config, &gpio);
```

Comment:
Perhaps this should be done in the start function?

Reply:
dup